docs/adminguide/stats_organization.rst

```diff
@@ -0,0 +1,23 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+.. _doc_adminguide_stats_organization:
+
+Extract organization statistics
+===============================
+
+In order to extract organization statistics, the administrator needs to provide :
+
+- The parsec metadata server location, through a parsec url ``parsec://hostname:port``
+- The ``administration_token`` configured in the parsec metadata server
+- The organization name
+
+.. code-block:: shell
+
+    $ parsec.cli core stats_organization --addr=parsec://example.com -T s3cr3t TestOrganization
```

Comment:
```suggestion
    $ parsec.cli core stats_organization --addr=parsec://example.com --administration-token=<ADMIN TOKEN> TestOrganization
```

Prefer `long` arguments instead of the short notation

Reply:
done.